integreat_cms/cms/migrations/0060_alter_languagetreenode.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.2.16 on 2023-01-30 16:08
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    """
+    Add field for en-/dis-abling machine translations to language tree model
+    """
+
+    dependencies = [
+        ("cms", "0059_add_poicategory_icon_color"),
```

Comment:
Could you rebase your branch again and rename this file to get a linear migration history?

Reply:
Oh whoops, of course. Is done.